packages/atomic/src/components/insight/atomic-insight-interface/atomic-insight-interface.pcss

```diff
@@ -0,0 +1,7 @@
+@import '../../../global/global.pcss';
+
+:host(.widget) {
```

Comment:
Instead of adding a class to the interface, which may be affected by CSS outside Atomic, perhaps this could be an attribute selector.

E.g.:
```css
:host([widget="true"]) {
  ...
}
```